.github/workflows/build-test-publish.yml

```diff
@@ -34,7 +34,11 @@ jobs:
       - name: Export Docker images from the Docker daemon
         env:
           BUILDER: ${{ matrix.builder }}
-        run: docker save ${{ matrix.builder }} heroku/heroku:${BUILDER##*-}-cnb | zstd -T0 --long=31 -o images.tar.zst
```

Comment:
This previous implementation for calculating the run image tag assumed the run image tag name used a naming pattern related to the directory name in this repository. 

Since the new builder image variant does not (the subdirectory is `salesforce-functions`, this has now had to be switched to looking up the run image from the builder.toml file itself - which is really a more correct solution anyway.